### PR TITLE
[Sage-755] Include Figma embed within Docs

### LINKED ISSUE #755: [Improvement] Include a Figma reference on all components pages
## Description

To create parity with Figma and the Design System, all components need Figma links or embedded design files.

### Figma Embed in a tab

tab-right

tab-open

--- 

### Scroll-to Embed 

Screen Shot 2021-08-30 at 4 15 28 PM

https://user-images.githubusercontent.com/24237393/131533817-e6638494-c022-44ab-a111-b19d71c35219.mp4

### PR DESCRIPTION
**Pull request by monicawheeler**
Closes #755 

## Description

Create a link to the Figma embed for all components available in the designs within Figma.

## Screenshots

|  Before  |  After  |
|--------|--------|
| Screen Shot 2021-09-01 at 1 32 44 PM | tab-open |

## Testing in `sage-lib`

1. Navigate to the [Alerts](http://localhost:4000/pages/component/alert)
2. Click the "Figma Designs" tab
3. Observe the embed within that tab content

## Testing in `kajabi-products`
1. (**LOW**) No impact on `kajabi-products`

### CODE REVIEW

**Review by anechol — APPROVED** [review]
Awesome! This will be super helpful going forward. Thanks for this work!

File: docs/app/helpers/components_helper.rb [hunk 1]
```diff
@@ -40,7 +94,8 @@ def sage_components
         rails: "done",
         react: "done",
         a11y: "done",
-        react_component_slug: "sage-card--card-highlight"
+        react_component_slug: "sage-card--card-highlight",
+        figma_embed: "https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2F9Km09NjlZHYWsMP7EGT8tI%2F%255BWIP%255D-Sage-3-%25E2%2580%2594-Admin-Components%3Fnode-id%3D6999%253A21623",
```
Comment (pixelflips) [inline docs/app/helpers/components_helper.rb:98]: This one looks like its embedding the breadcrumbs figma file.

File: docs/app/views/examples/components/_component.html.erb [hunk 2]
```diff
@@ -33,11 +33,15 @@
           {
             text: "Best Practices",
             target: "rules"
-          } 
+          },
+          {
+            text: "#{image_tag("logo-figma.svg", style: "margin-right: 4px", height: "18px")} Figma Designs",
+            target: "figma_embed"
+          },
```
Comment (pixelflips) [inline docs/app/views/examples/components/_component.html.erb:40]: Should we think about conditionally displaying this tab only if there is an embed available? I clicked a few that have no embeds which essentially shows as a blank page. Definitely not a show stopper.

**Review by pixelflips — APPROVED** [review]
Left a few minor comments. One incorrect embed link and a minor suggestion on when we display the Figma tab.

In addition, I noticed the Figma iframes don't resize on smaller screen sizes.
localhost_4000_pages_component_alert

Overall this is a great update and no major show-stoppers from me! 👍🏼

**Review by pixelflips — APPROVED** [review]
LGTM! 🔥  Great work!

One small oddity that we discussed is the odd default zoom behavior but seems to be an issue on Figma's side of things and not with this implementation. Could be looked into as a follow-up if nothing obvious comes up.

**Pull request merged**